core/query/src/main/java/datawave/core/query/configuration/GenericQueryConfiguration.java

```diff
@@ -70,6 +73,10 @@ public class GenericQueryConfiguration implements Serializable {
     // Whether or not this query emits every result or performs some kind of result reduction
     protected boolean reduceResults = false;
 
+    // either IMMEDIATE or EVENTUAL
+    private Map<String,ScannerBase.ConsistencyLevel> consistencyLevels = new HashMap<>();
```

Comment:
Should these properties be named to make it clear that this is TABLE to CONISITENCYLEVEL map. I was confused why it was a map until looking at other files in the pr.